src/model/Computation.js

```diff
@@ -157,4 +157,13 @@ export default class Computation extends Model {
 
 		this.dependencies = dependencies;
 	}
+
+	teardown () {
+		let i = this.dependencies.length;
+		while ( i-- ) {
+			if ( this.dependencies[i] ) this.dependencies[i].unregister( this );
+		}
+		if ( this.root.computations[this.key] === this ) delete this.root.computations[this.key];
```

Comment:
I used delete here because there are other sections that assume that all keys in computations have a value. Would it be better to fix those assumptions and just set undefined here? My understanding is delete is slow and can be a barrier to optimization.